### DISCUSSION TITLE
How to hide the locale in the URL path when I use the domain routing strategy?

### DISCUSSION THREAD

**Discussion by ozgur-shape** [vercel/next.js#48382] [Help]
### Summary

I am using [Domain Routing](https://nextjs.org/docs/advanced-features/i18n-routing#domain-routing) strategy to provide multiple locales for different domains. 

```js
module.exports = {
  i18n: {
    locales: ['en', 'de'],
    defaultLocale: 'en',
    domains: [
      {
        domain: 'example.com',
        defaultLocale: 'en',
      },
      {
        domain: 'example.de',
        defaultLocale: 'de',
      }
    ],
  },
}
```

In this case, users can access `DE` content by using the following URL;

- `example.com/de`

How can I disable this? Is there any way to use a strict locale value for a specific domain?

**Comment by levipadre** [comment]
I'm not 100% sure, but try localeDetection: false

**Reply by ozgur-shape** [reply]
Unfortunately, it doesn't work. Thanks for your help. 🙏

**Reply by levipadre** [reply]
I was facing the same issue, but I was able to solve today. Instead of next.config.js I used https://next-intl-docs.vercel.app/docs/next-13 and it worked for me.
Domain-based rounting info here: is https://next-intl-docs.vercel.app/docs/next-13/middleware#domain-based-routing

**Comment by thatgriffith** [comment]
I am stuck with the same problem. Did you manage to solve it? I am using next-i18next and react-i18next.
I am also not able to make the domain config to work in production with real domains. I am using Netlify as hosting service. Maybe the DNS config is incorrect. Have you manage to get it to work with real domains?
